EXILED/Exiled.API/Features/Roles/HumanRole.cs

```diff
@@ -32,19 +32,10 @@ internal HumanRole(HumanGameRole baseRole)
         /// <inheritdoc/>
         public override RoleTypeId Type => Base.RoleTypeId;
 
-        /// <summary>
-        /// Gets or sets the <see cref="SpawnableTeamType"/>.
-        /// </summary>
-        public SpawnableTeamType SpawnableTeamType
```

Comment:
You can make this into Team, there is a new thing attached to base afaik

Reply:
We already have Team property in role. No need to create another one